values.yaml

```diff
@@ -422,6 +422,9 @@ st2chatops:
   env:
     HUBOT_ADAPTER: slack
     HUBOT_SLACK_TOKEN: xoxb-CHANGE-ME-PLEASE
+  # If api_key is defined, then use this key instead of username/password for chatops to authenticate with StackStorm.
+  # Please make sure that this key gets added to st2.apikeys above so that chatops can use this key.
+  #api_key:
```

Comment:
Could the st2 API Key be configured via existing `env` definition?

```
env:
  HUBOT_ADAPTER: slack
  HUBOT_SLACK_TOKEN: xoxb-CHANGE-ME-PLEASE
  ST2_API_KEY: 123
```

Reply:
The problem is, I don't want ST2_AUTH_USERNAME or ST2_AUTH_PASSWORD exposed to the chatops process. So, adding the ST2_API_KEY var is only part of the issue.

Reply:
OK. I reworked this so ST2_API_KEY comes from `st2chatops.env` instead. That feels much cleaner. Thanks!